src/commands/libcoreChangePassword.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import { fromPromise } from 'rxjs/observable/fromPromise'
+
+import { createCommand, Command } from 'helpers/ipc'
+import { withLibcore } from '@ledgerhq/live-common/lib/libcore/access'
+
+type Input = {
+  oldPassword: string,
+  newPassword: string,
+}
+type Result = void
+
+const cmd: Command<Input, Result> = createCommand(
+  'libcoreChangePassword',
```

Comment:
do you blacklist this command to be logged in order to filter out the logs to contain the input object?